app/tests/integration/nimbus/conftest.py

```diff
@@ -143,10 +143,45 @@ def experiment_name(request):
     return f"{request.node.name[:75]}-{str(uuid.uuid4())[:4]}"
 
 
+@pytest.fixture(name="load_experiment_outcomes")
+def fixture_load_experiment_outcomes():
+    """Fixture to create a list of outcomes based on the current configs."""
+    outcomes = {"firefox_desktop": "", "fenix": "", "firefox_ios": ""}
+    base_path = "/code/app/experimenter/outcomes/jetstream-config-main/outcomes"
+
+    for k in list(outcomes):
+        outcomes[k] = [
+            name.split("_")[0].rsplit(".")[0]
+            for name in os.listdir(f"{base_path}/{k}")
+            if "example" not in name
+        ]
+    return outcomes
```

Comment:
Can't you just discover them from the list in the html?  Like active the drop down and just click wahtever the first option is?

Reply:
No. The Select element on the page isn't a "normal" select. It seems to be hidden within some react element ```id="react-select-126-input"```, and all of the CSS is munged on the page as well. I am open to thoughts on this, but I didn't think changing the page was worth the effort.

Reply:
Hrmm yeah that sucks.  Oh well.  We'll just have to do it this way 👍